apps/demo-material/src/styles.css

```
@@ -1,86 +1,316 @@
-.app-shell h2 {
-  font-size: 1.1rem;
-  font-weight: 600;
-  margin-top: 2rem;
-  margin-bottom: 0.75rem;
+.app-shell code {
+  padding: 0.1rem 0.45rem;
+  border: 1px solid rgba(86, 104, 138, 0.16);
+  border-radius: 999px;
+  background: rgba(255, 255, 255, 0.52);
+  font-family:
+    'SF Mono', 'SFMono-Regular', Consolas, 'Liberation Mono', monospace;
```

Stylelint (17.10.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)